README.md

```diff
@@ -534,16 +534,16 @@ For information about what changes are included in each release, please see the
 
 
 ## 17. Release History
-| Release Version | Release Date | Changes Included |
-| --------------- | ------------ | ---------------- |
-| 0.1.0           | 2020-03-05   | <ul><li>Initial release</li></ul>  |
+| Release Version | Release Date | Changes Included                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|-----------------|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
?

Reply:
Ah ! I think it is my markdown editor that added those lines mentioning compatibility issues. Would you want me to take the line off